packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "3.11.1",
+  "version": "3.11.2",
```

Comment:
Do we want to publish this PR immediately after merging?

Reply:
@bkjohnson I believe thats the game plan that @k80bowman and I chatted about during standup. It's that once this PR is merged in we'd do a release and then I could update our dependencies on the vets-website repo. With the dependencies updated  I could then run vets-website locally and in IE11 confirm that react-components are not broken